code/datums/lazy_template.dm

```diff
@@ -64,12 +64,21 @@
 	if(!reservation)
 		CRASH("Failed to reserve a block for lazy template: '[key]'")
 
+	// lists kept for overall loading
 	var/list/loaded_atom_movables = list()
 	var/list/loaded_turfs = list()
 	var/list/loaded_areas = list()
+
+	// list which matches a type to a list of things for that type
+	var/list/type_to_list = list(
```

Comment:
do we really need this level of extensibility here? has overhead in a pretty hot loop